### ISSUE TITLE
Fix `image-alt` violations 

### ISSUE THREAD

**Issue by DanielCliftonGuardian** [guardian/dotcom-rendering#15066]
WCAG: 2.0 A (1.1.1)
Affected: 8 stories
Fix: Add alt attribute to images, or role="presentation" for decorative images

Affected Components
Components/InteractiveLayoutAtom (1 story)
Components/SlotBodyEnd/BrazeEpics (7 stories)
